AllJoyn/Platform/BridgeHost/App.xaml.cs

```diff
@@ -0,0 +1,359 @@
+﻿//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+//
+//*********************************************************
+
+using System;
+using Windows.ApplicationModel.Activation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+using Windows.ApplicationModel.Background;
+using Windows.ApplicationModel.AppService;
+using Windows.Foundation.Collections;
+using System.Linq;
+using BridgeRT;
+using Windows.Storage;
+using Windows.ApplicationModel;
+using System.Text;
+using Windows.ApplicationModel.DataTransfer;
+using System.IO;
+using System.Runtime.Serialization;
+using System.Runtime.Serialization.Json;
+using System.Threading.Tasks;
+using System.Diagnostics;
+
+// The Blank Application template is documented at http://go.microsoft.com/fwlink/?LinkId=402347&clcid=0x409
+
+namespace OpenT2T.Bridge
+{
+    [DataContract]
+    internal class DeviceProps
```

Comment:
>DeviceProps [](start = 19, length = 11)

Shouldn't this be a more dynamic JSON object that might have different properties for different onboarding implementations?